imagetest/test_suites/mdsmtls/mds_mtls_test.go

```diff
@@ -72,9 +86,54 @@ func checkCredsPresentWindows(t *testing.T) {
 }
 
 func TestMTLSCredsExists(t *testing.T) {
+	checkMTLSAvailable(t)
+	ctx := utils.Context(t)
+	var rootKeyFile string
 	if utils.IsWindows() {
+		rootKeyFile = filepath.Join(os.Getenv("ProgramData"), "Google", "Compute Engine", "mds-mtls-root.crt")
 		checkCredsPresentWindows(t)
 	} else {
+		rootKeyFile = filepath.Join("run", "google-mds-mtls", "root.crt")
 		checkCredsPresent(t)
 	}
+	rootKey, err := ioutil.ReadFile(rootKeyFile)
+	if err != nil {
+		t.Fatal(err)
+	}
+	caCertPool := x509.NewCertPool()
+	caCertPool.AppendCertsFromPEM(rootKey)
+	client := http.Client{
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{
+				RootCAs: caCertPool,
```

Comment:
I think client credentials `mds-mtls-client.key`/`client.key` is missing here

Reply:
Ah, right. I think the new code should work here.